### PR TITLE
Fix duplicated video loading issue and inconsistent footer's social media icon.

### PR DESCRIPTION
**Pull request by hoangviet1993**
- [x] Only 1 video is loaded on page load. If viewport is resized, 2nd video is loaded in.
- [x] Fix footer social media icon size (Somehow all icons except facebook got smaller, not sure when did that happen)

Changes are live at:
https://oppia-foundation-test-server.appspot.com

### CODE REVIEW

**Comment by hoangviet1993** [comment]
@dawsoneliasen Can you help verify if the duplicated video issue is fixed on https://oppia-foundation-test-server.appspot.com?

File: app/static/pages/app.js [hunk 1]
```diff
@@ -13,8 +13,8 @@
 // limitations under the License.
 
 var oppiaFoundationWebsite = angular.module(
-  'oppiaFoundationWebsite', ['ngMaterial', 'ngMessages', 'ngRoute', 'duScroll',
-    'ui.bootstrap', 'headroom']);
+  'oppiaFoundationWebsite', ['angular-matchmedia', 'ngMaterial', 'ngMessages',
```
Comment (seanlip) [inline app/static/pages/app.js]: I don't think you need to add a 3rd-party lib for matchmedia -- just make a small service on our end, similar to what we do in oppia (windowDimensionsService).

Comment (hoangviet1993) [reply app/static/pages/app.js]: Yep, I agree with not using a 3rd party lib. Replaced 3rd party angular-matchmedia with WindowDimensionsService!

File: app/app.yaml [hunk 2]
```diff
@@ -21,6 +21,13 @@ handlers:
   http_headers:
     Cache-Control: 'public, max-age=600'
 
+- url: /services
```
[the remark this answers is not in the document]
Comment (hoangviet1993) [reply app/app.yaml]: Done!

File: app/static/pages/services/WindowDimensionsService.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      getWidth: function() {
+        return (
+          $window.innerWidth || document.documentElement.clientWidth ||
+        document.body.clientWidth);
+      },
+      registerOnResizeHook: function(hookFn) {
+        onResizeHooks.push(hookFn);
+      },
+      isDesktopViewWidth: function() {
+        var DESKTOP_VIEWWIDTH_PX = 1279;
```
Comment (seanlip) [inline app/static/pages/services/WindowDimensionsService.js]: If VIEW WIDTH is two words then VIEWWIDTH --> VIEW_WIDTH, here and below.

Comment (seanlip) [reply app/static/pages/services/WindowDimensionsService.js]: Also, suggest something like MIN_DESKTOP_VIEW_WIDTH_PX or DESKTOP_WIDTH_THRESHOLD_PX (to give the idea of a boundary/threshold).

Comment (hoangviet1993) [reply app/static/pages/services/WindowDimensionsService.js]: I went with DESKTOP_WIDTH_THRESHOLD_PX. Thanks :)

**Pull request merged**